mmdet/models/dense_heads/fcos_head.py

```diff
@@ -80,26 +126,31 @@ def _init_layers(self):
         self.reg_convs = nn.ModuleList()
         for i in range(self.stacked_convs):
             chn = self.in_channels if i == 0 else self.feat_channels
+            if self.dcn_on_last_conv and \
+                    i == self.stacked_convs - 1:
```

Comment:
Use parentheses instead of back slashes.
`if (self.dcn_on_last_conv and i == self.stacked_convs - 1)` and let yapf to format it.